include/repository.php

```diff
@@ -736,25 +743,28 @@ function get_github_api_url($github_url)
     return $api_url;
 }
 
-function post_github_pull_request_comment($projectid, $pull_request, $comment, $cdash_url)
+function post_github_pull_request_comment($project, $pull_request, $comment, $cdash_url)
```

Comment:
What are your thoughts on type hinting? I would have hinted the `Project` here, but because we're still supporting php 5.4 (?) we're still unable to type hint scalars.